### ISSUE TITLE
Stack traces for Jasmine2 don't include asynchronous events

### ISSUE THREAD

**Issue by juliemr** [angular/jasminewd#15]
None of the ===async task=== stuff is getting added to the reported stack traces.

**Comment by juliemr** [comment]
Specifically, this is when an error is thrown, such as `element(by.binding('nonexistant')).click()`

**Comment by juliemr** [comment]
See also jasmine/jasmine#734

**Comment by donaldpipowitch** [comment]
Note sure if this is related but, when I use Jasmine 2 I get stack traces like this: 

````
  Message:
    Failed: undefined is not a function
  Stack:
    Error: Failed: undefined is not a function
        at /my-app/node_modules/protractor/node_modules/jasminewd2/index.js:104:16
````

While Jasmine 1 has this stack trace:

````
   Message:
     TypeError: undefined is not a function
   Stacktrace:
     TypeError: undefined is not a function
    at [object Object].<anonymous> (/my-app/src/file_e2e.js:31:29)
````

**Comment by juliemr** [comment]
This is fixed with the upgrade to 2.3.1
